# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/magic/mut-command-list.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- a/textio/txMain.c	2025-08-20 00:59:26.938115380 +0200
+++ b/textio/txMain.c	2025-08-20 00:59:57.035113940 +0200
@@ -61,7 +61,7 @@
 
 /* The readline completion function requires a command list containing	*/
 /* just the command name (without the accompanying help text line)	*/
-extern const char **magic_command_list;
+extern char **magic_command_list;
 
 #endif
 
